aksel.nav.no/website/pages/eksempler/dialog/trap-focus.tsx

```diff
@@ -0,0 +1,226 @@
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    onSave({ ...person, role, department });
+  };
+
+  return (
+    <>
+      <Dialog.Header withClosebutton={false}>
+        <HStack justify="space-between" align="center">
+          <Dialog.Title>
+            Rediger {person.firstName} {person.lastName}
+          </Dialog.Title>
+          <HStack gap="space-4">
+            <Button
+              variant="tertiary"
+              size="small"
+              icon={<ChevronUpIcon title="Forrige person" />}
+              data-color="neutral"
+              type="button"
+              onClick={onPrevious}
+              disabled={!onPrevious}
+            />
+            <Button
+              variant="tertiary"
+              size="small"
+              icon={<ChevronDownIcon title="Neste person" />}
+              data-color="neutral"
+              type="button"
+              onClick={onNext}
+              disabled={!onNext}
+            />
+            <Dialog.CloseTrigger>
+              <Button
+                variant="tertiary"
+                size="small"
+                icon={<XMarkIcon title="Lukk dialog" />}
+                data-color="neutral"
+                type="button"
+              />
+            </Dialog.CloseTrigger>
+          </HStack>
+        </HStack>
+      </Dialog.Header>
+      <Dialog.Body>
+        <form id="form" onSubmit={handleSubmit}>
+          <VStack gap="space-16">
+            <TextField
+              label="Rolle"
+              value={role}
+              onChange={(e) => setRole(e.target.value)}
+            />
+            <TextField
+              label="Avdeling"
+              value={department}
+              onChange={(e) => setDepartment(e.target.value)}
+            />
+          </VStack>
+        </form>
+      </Dialog.Body>
+      <Dialog.Footer>
+        <Dialog.CloseTrigger>
+          <Button variant="secondary">Avbryt</Button>
+        </Dialog.CloseTrigger>
+        <Button form="form">Lagre</Button>
+      </Dialog.Footer>
+    </>
+  );
+}
+
+interface Person {
+  id: number;
+  firstName: string;
+  lastName: string;
+  role: string;
+  department: string;
+}
+
+const initialPeople: Person[] = [
+  {
+    id: 1,
+    firstName: "Jean-Luc",
+    lastName: "Picard",
+    role: "Kaptein",
+    department: "Bro",
+  },
+  {
+    id: 2,
+    firstName: "William",
+    lastName: "Riker",
+    role: "Kommandør",
+    department: "Bro",
+  },
+  {
+    id: 3,
+    firstName: "Geordi",
+    lastName: "La Forge",
+    role: "Sjefsingeniør",
+    department: "Maskinrom",
+  },
+];
+
+// EXAMPLES DO NOT INCLUDE CONTENT BELOW THIS LINE
+export default withDsExample(Example);
+
+/* Storybook story */
+export const Demo = {
+  render: Example,
+};
+
+export const args = {
+  index: 12,
+  desc: "'trap-focus' modal dialog lar brukeren navigere innenfor dialogen med pointer/touch uten å miste 'focus'.",
```

Comment:
```suggestion
  desc: '`modal="trap-focus"` lar brukeren interagere med innholdet utenfor dialogen med pointer/touch, mens fokus forblir innenfor dialogen.',
```

Should we also say something about when this should (not) be used? (We mention "expert systems" in the docs.)

Should we mention that the content behind the dialog will not be available with keyboard or screen reader?